test/regress/run_regression

```diff
@@ -49,6 +49,16 @@ cvc4=$1
 benchmark_orig=$2
 benchmark="$benchmark_orig"
 
+if [ ! -z "$TEST_GROUP" ]; then
```

Comment:
Instead of 'if [ ! -z  "$TEST_GROUP" ]' you could also use 'if [ -n "$TEST_GROUP" ]'.